MapboxCoreNavigationTests/NavigationEventsManagerTests.swift

```diff
@@ -53,9 +53,9 @@ class NavigationEventsManagerTests: XCTestCase {
         
         XCTAssertEqual(events.count, 3, "There should be one depart, one reroute, and one arrive event.")
         
-        let departEvent = events.filter { $0.event == MMEEventTypeNavigationDepart }.first!
-        let rerouteEvent = events.filter { $0.event == MMEEventTypeNavigationReroute }.first!
-        let arriveEvent = events.filter { $0.event == MMEEventTypeNavigationArrive }.first!
+        guard let departEvent = events.filter({ $0.event == MMEEventTypeNavigationDepart }).first else { XCTFail(); return }
```

Comment:
nit: add description for `XCTFail` calls?

Reply:
There is a possibility that this test might slightly change, will check afterwards.